docs/apis/experimental.md

```
@@ -25,3 +25,15 @@ This namespace contains experimental features. These are under development, and
 .. automodule:: experimental.continuous_space.continuous_space_agents
    :members:
 ```
+
+## Continuous Space
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)